staging-service/src/main/java/com/github/cafdataprocessing/services/staging/utils/JsonMinifier.java

```diff
@@ -103,7 +103,7 @@ private static void processJsonTokens(final JsonParser parser,
                                           final int fieldValueSizeThreshold,
                                           final Map<String, String> binaryFilesUploaded) throws IOException, InvalidBatchException
     {
-        String dataBuffer = null;
+        String dataBuffer = "";
```

Comment:
Maybe add **dataBuffer != null &&** on line 159

Does  line 188 need to be updated too? **dataBuffer = "";**

Reply:
You're right, less change and the code should be more efficient that check if contains("").